### PR TITLE
Add DM_DEVICE_CERTIFICATE Table into SQL Scripts

### PR DESCRIPTION
**Pull request by dilee**
Adding DM_DEVICE_CERTIFICATE Table into SQL Scripts.

### CODE REVIEW

File: features/device-mgt/org.wso2.carbon.device.mgt.server.feature/src/main/resources/dbscripts/cdm/mssql.sql [hunk 1]
```diff
@@ -1,9 +1,16 @@
 CREATE TABLE DM_DEVICE_TYPE (
-     ID INT IDENTITY NOT NULL,
+     ID INTEGER IDENTITY NOT NULL,
      NAME VARCHAR(300) DEFAULT NULL,
      PRIMARY KEY (ID)
 );
 
+CREATE TABLE DM_DEVICE_CERTIFICATE (
+      ID                    INTEGER IDENTITY NOT NULL,
```
Comment (prabushi) [inline features/device-mgt/org.wso2.carbon.device.mgt.server.feature/src/main/resources/dbscripts/cdm/mssql.sql:8]: Fix formatting.

File: features/device-mgt/org.wso2.carbon.device.mgt.server.feature/src/main/resources/dbscripts/cdm/mysql.sql [hunk 2]
```diff
@@ -4,6 +4,12 @@ CREATE TABLE IF NOT EXISTS DM_DEVICE_TYPE (
      PRIMARY KEY (ID)
 )ENGINE = InnoDB;
 
+CREATE TABLE IF NOT EXISTS DM_DEVICE_CERTIFICATE (
+      ID                    INTEGER auto_increment NOT NULL,
```
Comment (prabushi) [inline features/device-mgt/org.wso2.carbon.device.mgt.server.feature/src/main/resources/dbscripts/cdm/mysql.sql:8]: Fix formatting.

File: features/device-mgt/org.wso2.carbon.device.mgt.server.feature/src/main/resources/dbscripts/cdm/mssql.sql (hunk 1 above)
Comment (dilee) [reply features/device-mgt/org.wso2.carbon.device.mgt.server.feature/src/main/resources/dbscripts/cdm/mssql.sql:8]: Fixed in https://github.com/wso2/carbon-device-mgt/commit/99ff4cbc10dbc82d5dd7dfed72ce6f1bb11cf3e1

File: features/device-mgt/org.wso2.carbon.device.mgt.server.feature/src/main/resources/dbscripts/cdm/mysql.sql (hunk 2 above)
Comment (dilee) [reply features/device-mgt/org.wso2.carbon.device.mgt.server.feature/src/main/resources/dbscripts/cdm/mysql.sql:8]: Fixed in https://github.com/wso2/carbon-device-mgt/commit/c161d7b65941c2afd6e24b80ba64f641d75f5351

**Pull request merged**